package.json

```diff
@@ -28,43 +29,42 @@
+    "start": "yarn clean-dist && webpack serve --config webpack.dev.mjs --env dev --progress --node-env development",
     "content": "node src/scripts/build-content-tree.mjs ./src/content ./src/_content.json",
     "bundle-analyze": "run-s clean fetch content && webpack --config webpack.prod.mjs --node-env production && run-s printable content && webpack --config webpack.ssg.mjs --node-env production --env ssg --profile --json > stats.json && webpack-bundle-analyzer stats.json",
     "fetch-repos": "node src/utilities/fetch-package-repos.mjs",
     "fetch": "run-p fetch:*",
     "fetch:readmes": "node src/utilities/fetch-package-readmes.mjs",
     "fetch:supporters": "node src/utilities/fetch-supporters.mjs",
-    "prebuild": "npm run clean",
+    "prebuild": "yarn clean",
     "build": "run-s fetch-repos fetch content && webpack --config webpack.prod.mjs --node-env production && run-s printable content && webpack --config webpack.ssg.mjs --node-env production --env ssg",
-    "postbuild": "npm run sitemap",
-    "build-test": "npm run build && http-server --port 4200 dist/",
+    "postbuild": "yarn sitemap",
+    "build-test": "yarn build && http-server --port 4200 dist/",
     "serve-dist": "http-server --port 4200 dist/",
-    "test": "npm run lint",
+    "test": "yarn lint",
     "lint": "run-s lint:*",
-    "lint:js": "npm run lint-js .",
+    "lint:js": "yarn lint-js .",
     "lint-js": "eslint --cache --cache-location .cache/.eslintcache",
-    "lint:markdown": "npm run lint-markdown *.md",
+    "lint:markdown": "yarn lint-markdown *.md",
     "lint-markdown": "markdownlint --config ./.markdownlint.json --ignore '.vale/**/*.md' --ignore '.github/**/*.md'",
     "lint:prose": "vale --config='.vale.ini' src/content",
     "lint:links": "hyperlink -c 8 --root dist -r dist/index.html --canonicalroot https://webpack.js.org/ --internal --skip /plugins/extract-text-webpack-plugin/ --skip /printable --skip https:// --skip http:// --skip sw.js > internal-links.tap; cat internal-links.tap | tap-spot",
     "sitemap": "cd dist && sitemap-static --ignore-file=../sitemap-ignore.json --pretty --prefix=https://webpack.js.org/ > sitemap.xml",
-    "serve": "npm run build && sirv start ./dist --port 4000",
-    "preprintable": "npm run clean-printable",
+    "serve": "yarn build && sirv start ./dist --port 4000",
+    "preprintable": "yarn clean-printable",
     "printable": "node ./src/scripts/concatenate-docs.mjs",
     "jest": "NODE_OPTIONS=--experimental-vm-modules jest --config=jest.config.mjs",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "prettier": "prettier --write '**/*.{js,json,jsx,css,scss,md,mdx}'",
-    "prepare": "husky install",
-    "postinstall": "rimraf ./node_modules/.cache/webpack"
+    "prepare": "husky install && rimraf ./node_modules/.cache/webpack && yarn-deduplicate --strategy fewer"
```

Comment:
I think adding this line along with the version info should be enough, and there's no need to update those in markdown files. I'd like to make this project compatible with npm as much as possible since it's the default one bundled with Node though we do have some yarn specific features at the moment.